eslint.config.mjs

```diff
@@ -48,31 +49,12 @@ export default [
       globals: {
         ...globals.browser,
         ...globals.node,
-        fixture: true,
-        test: true,
-        describe: true,
-        it: true,
-        expect: true,
-        beforeEach: true,
-        afterEach: true,
-        beforeAll: true,
-        afterAll: true,
         vi: true,
       },
     },
     rules: {
       "unused-imports/no-unused-imports": "error",
       "unused-imports/no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
-      "ban/ban": [
```

Comment:
I removed the ban. I think it was used for Jasmine tests. Now we no longer have those. There is a vitest plugin (which we are already using) that will show warnings when you focus a test.